src/ducks/account/LoanDetailsPage.jsx

```diff
@@ -1,11 +1,260 @@
 import React from 'react'
 import AccountDetailsHeader from 'ducks/account/AccountDetailsHeader'
+import { translate } from 'cozy-ui/transpiled/react/I18n'
+import CompositeRow from 'cozy-ui/transpiled/react/CompositeRow'
+import { Figure } from 'components/Figure'
+import { GROUP_DOCTYPE, ACCOUNT_DOCTYPE } from 'doctypes'
+import { get } from 'lodash'
+import { Section as BaseSection } from 'components/Section'
+import cx from 'classnames'
+import styles from 'ducks/account/LoanDetailsPage.styl'
+
+const DATE_FORMAT = 'DD/MM/YY'
+
+const Row = translate()(props => {
+  const {
+    type,
+    title,
+    value: originalValue,
+    caption,
+    t, // eslint-disable-line no-unused-vars
+    f,
+    className,
+    ...rest
+  } = props
+
+  if (!originalValue) {
+    return null
+  }
+
+  let value
+
+  switch (type) {
+    case 'amount':
+      value = <Figure total={originalValue} symbol="€" />
+      break
+
+    case 'date':
+      value = f(originalValue, DATE_FORMAT)
+      break
+
+    case 'rate':
+      value = originalValue + '%'
+      break
+
+    case 'nb-payments':
+      value = 'x' + originalValue
+      break
+
+    default:
+      value = originalValue
+      break
+  }
+
+  const right = <span className={styles.LoanDetailsRow__value}>{value}</span>
+
+  return (
+    <div className={cx(styles.LoanDetailsRow, className)}>
+      <CompositeRow
+        className={cx(styles.LoanDetailsRow__inner)}
+        primaryText={title}
+        secondaryText={caption}
+        right={right}
+        {...rest}
+      />
+    </div>
+  )
+})
+
+const Section = props => {
+  const { title, children } = props
+
+  return <BaseSection title={title}>{children}</BaseSection>
+}
+
+const hasInfos = (account, paths) => {
```

Comment:
At first I thought you need all the infos to return true. What do you think of renaming the function to hasSomeInfos to make it clearer that only 1 piece of info is necessary.

Reply:
`return _.some(paths.map(path => get(account, path))` ;)